src/psyclone/tests/lfric_build.py

```diff
@@ -98,6 +98,7 @@ class LFRicBuild(Compile):
                               "field_mod",
                               "integer_field_mod",
                               "r_solver_field_mod",
+                              "r_tran_field_mod",
```

Comment:
Not something for this PR but it does beg the question "why don't we just use the Makefile to build the infrastructure?". Can you or @hiker remember why we don't do that?

Reply:
I can't, unfortunately. Perhaps @hiker can?